doc/mgmt_netconf/hld_NETCONF-NTT.md

```diff
@@ -0,0 +1,790 @@
+
+NETCONF server must comply to below standards. (excluding capabilities not supported in tihs release)
+
+<!-- omit in toc -->
+##### Table: Supported standards
+| Feature      | Compliant Standard                      |
+| ------------ | --------------------------------------- |
+| NETCONF      | RFC6241 (base:1.1), RFC4741 (base:1.0)  |
+| YANG         | RFC7950 (YANG 1.1), RFC 6020 (YANG 1.0) |
+| OpenApi Spec | 3.0                                     |
+
+### Supported capabilities
+
+NETCONF server must support capability marked "Supported" == "YES" or Partially".
+
+<!-- omit in toc -->
+##### Table: Supported capabilities
+| Capability Name                    | Supported | Remark                                     |
+| ---------------------------------- | --------- | ------------------------------------------ |
+| Writable-Running Capability        | Yes       | RFC6241 8.2                                |
+| Candidate Configuration Capability | No        | RFC6241 8.3 [*1]                           |
+| Confirmed Commit Capability        | No        | RFC6241 8.4 [*1]                           |
+| Rollback-on-Error Capability       | Yes       | RFC6241 8.5                                |
+| Validate Capability                | Yes       | RFC6241 8.6 [*2]                           |
+| Distinct Startup Capability        | No        | RFC6241 8.7 [*3]                           |
+| URL Capability                     | Partially | RFC6241 8.8 only for edit-config operation |
+| XPath Capability                   | Yes       | RFC6241 8.9                                |
+
+- [*1] As SONiC management framework doesn’t support transaction for configuration update, this capability is not supported
+- [*2] Syntactic and semantic validation for payload will be supported, but explicit "validation" operation will not be supported in this release
+- [*3] As SONiC infrastructure doesn’t have separate Startup and Running Redis DB, this capability is not supported
+
+### Supported operations
+
+NETCONF server must support operations marked "Supported == YES or Partially".
+
+<!-- omit in toc -->
+##### Table: Supported operations
+| RFC Operation Name | Supported | Remark |
+| ------------------ | --------- | ------ |
+| edit-config        | Partially | [*1]   |
+| get-config         | Partially | [*2]   |
+| get                | No        |        |
+| copy-config        | No        | [*3]   |
+| delete-config      | No        | [*3]   |
+| lock               | No        | [*4]   |
+| unlock             | No        | [*4]   |
+| close-session      | Yes       |        |
+| kill-session       | Yes       |        |
+
+- [*1] only below options are supported:
+  - operation : merge,delete
+  - target : running (as commit feature is not supported, only Running config is applicable)
+  - test-option : test-and-set
+  - error-option : rollback-on-error
+- [*2] get-config will return config in netopeer2/sysrepo and not from CONFIG_DB
```

Comment:
It is understood that the sysrepo serves running datastore. But, I am not clear on how to init the sysrepo at the server init. Can you elaborate?

Reply:
As of this release, sysrepo will serve as running database only for the config modified by NETCONF. Thus, it will be initialized with no data on startup. In future, we are planning to introduce functionality to sync sysrepo with CONFIG_DB but that is not scope of this release.
Would this answer your question? Let me know if you were asking from different aspect. (I will update HLD doc otherwise)